app/src/main/java/org/asteroidos/sync/ble/ScreenshotService.java

```diff
@@ -185,6 +188,63 @@ public void onEvent(ReadWriteEvent e) {
             Log.e("ScreenshotService", e.status().toString());
     }
 
+    private Uri createFile(byte[] totalData) throws IOException {
+        String dirStr = Environment.getExternalStoragePublicDirectory(Environment.DIRECTORY_PICTURES) + "/AsteroidOSSync";
+        Uri uri;
+
+        String timeStamp = new SimpleDateFormat("yyyyMMdd_HHmmss", Locale.US).format(System.currentTimeMillis());
+        File fileName;
+        fileName= new File(dirStr + "/Screenshot_" + timeStamp + ".jpg");
```

Comment:
nit: Could you add a space before that = ? And maybe bring it on the same line as the previous one?